Cart/cart.css

```diff
@@ -0,0 +1,129 @@
+.cart-title{
+    display: flex;
+    justify-content: center;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+.cart-container{
+    display: grid;
+    grid-template-areas: "products checkout";
+    grid-template-columns: 1fr 0.2fr;
+    margin-left: 14rem;
+    margin-right: 14rem;
+    
+}
+
+.products{
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    /* margin: 3rem; */
+    margin-top: 0rem;
+}
+.product-card-horizontal{
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    width: 31rem;
+    height: 15rem;
+    background-color: var(--color-white);
+    border-radius: 1rem;
+    justify-content: center;
+    grid-area: products;
+}
+
+.icon-on-horizontal-card{
+    position: absolute;
+    right: 0;
+    background-color: transparent;
+    color: var(--color-red);
+    font-size: larger;
+    cursor: pointer;
+}
+
+.image-section{
+    position: relative;
+    align-self: center;
+    background-color: var(--color-white);
+}
+.horizontal-card-details{
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-white);
+    border-radius: 1rem;
+    margin-right: 1rem;
+    justify-content: center;
+}
+
+.qty-section{
+    display: inline-block;
+    background-color: var(--color-white);
+    margin-left: 1rem;
+}
+
+.qty-number{
```

Comment:
Keep consistent spacing.
The rest of the code looks good, class names are descriptive and self-explanatory.